controllers/discovery/initDiscoveryDomainData.js

```diff
@@ -45,11 +51,21 @@ const InitDiscoveryDomainData = async (req, res) => {
     });
     let suggestedDomains = domainWithCommonFollowerResult;
 
+    let totalData = await sequelize.query(totalDataQuery, {
+      type: QueryTypes.SELECT,
+      replacements: {
+        userId
+      },
+      raw: true
+    });
+    totalData = totalData?.[0]?.total_data || 0;
+
     return res.status(200).json({
       success: true,
       message: `Fetch discovery data success`,
       suggestedDomains,
-      page: page + 1
+      page: page + 1,
+      total_page: totalData > 0 ? Math.ceil(totalData / limit) : 0
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The calculation for total pages is done inline within the response object. For better readability and maintainability, consider moving this calculation to its own line before the response object.
```diff
- total_page: totalData > 0 ? Math.ceil(totalData / limit) : 0
+ let totalPages = totalData > 0 ? Math.ceil(totalData / limit) : 0;
...
+ total_page: totalPages
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->